provider/cmd/pulumi-resource-purrl/schema.json

```diff
@@ -131,7 +145,7 @@
                     "type": "string"
                 },
                 "responseCodes": {
-                    "description": "The expected response code.",
+                    "description": "The expected response code(s). Deprecated -- use `expectedResponseCodes` instead.",
```

Comment:
Hi @necaris,

can you use `deprecationMessage` instead?

see  https://www.pulumi.com/docs/guides/pulumi-packages/schema/#property